code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1585,3 +1593,68 @@
 	clot_rate = 0.4 //slightly better than regular coagulant
 	passive_bleed_modifier = 0.5
 	overdose_threshold = 10 //but easier to overdose on
+
+/datum/reagent/medicine/bruvital
+	name = "bruvital"
+	description = "An experimental and secretative chemical, used to rapidly repair brute damage. Overdose causes extreme, rapid physical degradation."
+	reagent_state = LIQUID
+	color = "#C8A5DC"
+	metabolization_rate = 2 * REAGENTS_METABOLISM
+	overdose_threshold = 30
+
+/datum/reagent/medicine/bruvital/on_mob_life(mob/living/carbon/M, delta_time, times_fired)
+	M.adjustBruteLoss(-10 * REM * delta_time, 0)
+	..()
+	. = TRUE
```

Comment:
im not sure whatever the fuck return value means for chems, but put this at the start, and make these procs use descript argument names rather than single letters

Reply:
Gonna just replace these wack chems with c3's